core/src/main/kotlin/org/evomaster/core/problem/externalservice/httpws/service/HarvestActualHttpWsResponseHandler.kt

```diff
@@ -479,30 +480,36 @@ class HarvestActualHttpWsResponseHandler {
         return out
     }
 
-    private fun getURL(url: String): List<String> {
-        return url.split("::")
+    /**
+     * A request description contains information about the method, url, headers, and body.
+     * This extract the URL information as [URL] for ease of use.
+     *
+     * e.g.: GET::http://exists.local:12354/api/fzz::[]::{none}
+     */
+    private fun getURLFromRequestDescription(url: String): URL {
```

Comment:
i see... that is the confusing part: the input `url` has a misleading name, as it is not an URL, but rather a "request description". change name of all these inputs `url` string objects (also in following methods)